server/channels/app/plugin_api_test.go

```diff
@@ -813,43 +813,12 @@ func TestPluginAPISavePluginConfig(t *testing.T) {
 	assert.Equal(t, expectedConfiguration, savedConfiguration)
 }
 
-func TestPluginAPIGetPluginConfig(t *testing.T) {
```

Comment:
`TestPluginAPIGetPluginConfig` failed because it doesn't create a proper plugin; it is only set up in a plugin API. Hence, I moved it into `TestPluginAPILoadPluginConfiguration`, which fixes the test.